app/src/main/java/org/matrix/matrixandroidsdk/gcm/RegisterGcmPusherTask.java

```diff
@@ -0,0 +1,22 @@
+package org.matrix.matrixandroidsdk.gcm;
+
+import android.content.Context;
+import android.os.AsyncTask;
+
+/**
+ * Created by leon on 2/28/15.
+ */
+class RegisterGcmPusherTask extends AsyncTask<Void, Void, Void> {
+
+    private Context mContext;
+
+    RegisterGcmPusherTask(Context context) {
+        mContext = context;
+    }
+
+    @Override
+    protected Void doInBackground(Void... voids) {
+
+        return null;
```

Comment:
What is this class doing? If it's a stub, please add a ``TODO`` stating what needs to be done here.